app/konnect-platform/self-managed-migration.md

```diff
@@ -0,0 +1,269 @@
+
+To migrate plugins from a self-managed deployment to {{site.konnect_short_name}}, review 
+[Konnect Compatibility page](/konnect-compatibility/#plugin-compatibility) to check for supported and unsupported plugins.
+Also review any plugin configuration values, as certain values are unsupported in {{site.konnect_short_name}} and may require additional
+changes to your configuration.
+
+### Custom plugins
+
+{{site.konnect_short_name}} supports custom plugins with similar restrictions to pre-built plugins. Since {{site.konnect_short_name}} runs in a hybrid deployment mode, custom plugins can't access a database directly
+and can't provide custom Admin API endpoints. See the {{site.konnect_short_name}} documentation for more details
+on [custom plugin support](/gateway/entities/plugin/#custom-plugins) requirements.
+
+Migrating custom plugins to {{site.konnect_short_name}} requires uploading and associating your custom plugin's `schema.lua` file to 
+the desired Control Plane. This can be done using the 
+Konnect UI in [Gateway Manager](https://cloud.konghq.com/gateway-manager/) or the 
+[Konnect Control Planes Config API](/api/konnect/control-planes-config/v2/#/operations/list-plugin-schemas).
+
+Just like in self-managed deployments, the custom plugin code must be distributed to the Data Plane instances.
+
+{:.info}
+> **Note**: {{site.konnect_short_name}}'s Dedicated Cloud Gateways can support custom plugins
+but currently require a manual deployment process involving {{site.base_gateway}}'s support team. 
+Contact your Kong representative for more information.
+
+### {{site.base_gateway}} configuration
+
+We recommend migrating {{site.base_gateway}} configuration to {{site.konnect_short_name}} 
+using [decK](/deck/gateway/konnect-configuration/), the declarative management tool for {{site.base_gateway}}
+configurations.
+
+The general process for migrating the configuration involves "dumping" your existing  self-managed configuration
+to a local file, modifying the configuration slightly to remove any Workspace-specific metadata,
+and then synchronizing the configuration to your desired Control Plane in {{site.konnect_short_name}}.
+
+For example, if you have three Workspaces (`default`, `inventory`, and `sales`), use decK to dump the configuration of each Workspace to a local file:
+
+```sh
+deck gateway dump --workspace default --output-file default.yaml
+deck gateway dump --workspace inventory --output-file inventory.yaml
+deck gateway dump --workspace sales --output-file sales.yaml
+```
+
+When using `deck` to dump the configuration, the output file will include the Workspace name in the configuration.
+To remove the `_workspace` key from the configuration, you can use the `yq` tool:
+
+```sh
+yq -i 'del(._workspace)' default.yaml
+yq -i 'del(._workspace)' inventory.yaml
+yq -i 'del(._workspace)' sales.yaml 
+```
```

Comment:
```suggestion
When using `deck` to dump the configuration, the output file will include the Workspace name in the configuration.
```
I don't think this is that important, and it forces a user to download `yq` or figure out what it is.